client/extensions/woocommerce/state/action-list/selectors.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Get the current step in the action list.
+ *
+ * @param {Object} rootState The root Calypso state.
+ * @return {Array|null} The action list, or null if no action list is present.
+ */
+export function getActionList( rootState ) {
+	const woocommerce = rootState.extensions.woocommerce;
+	return get( woocommerce, 'actionList', null );
+}
+
+/**
+ * Get the current step in the action list.
+ *
+ * This selector returns the index of the step that is currently executing in the action list.
+ * (The first step in the list that does not have an endTime)
+ *
+ * @param {Object} actionList The action list to check.
+ * @return {Number|null} The index of the current step, or actionList.length if all steps are complete,
+ */
+export function getCurrentStepIndex( actionList ) {
+	for ( let i = 0; i < actionList.length; i++ ) {
```

Comment:
Why is actionList an argument? Shouldn't this always be the action list in state?